spec/features/user_can_see_which_users_have_liked_post_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.feature "See which users have liked", type: :feature do
+
+  before(:each) do
+    @user1 = User.create(first_name: "test_first_name", last_name: "test_last_name", email: "test_email@keepingitrails.com", password: "pa55w0rd")
+    @user2 = User.create(first_name: "test_second_name", last_name: "test_last_name", email: "test_email2@keepingitrails.com", password: "pa55w0rd")
+    @post = Post.create(user_id: @user1.id, message: "Test message")
+  end
+
+  scenario "A user can see his/her like on own post" do
+    successful_sign_in
+    click_button "Like"
+    click_link "Likes:"
```

Comment:
Note - if you take the semi-colon off likes as suggested in my comment above, take it off here as well (and below) to make sure tests pass.